### PR TITLE
Test director secrets pr

### PR DESCRIPTION
**Pull request by milanhajek**
You don't need to check controller.yaml file - is downloaded

### CODE REVIEW

**Review by chriswldenyer — APPROVED** [review]
LGTM

File: src/tools/extended-test-framework/scripts/deploy.sh [hunk 1]
```diff
@@ -96,6 +96,7 @@ if [ "${OPERATION}" == "delete" ]; then
   kubectl delete configmap tc-config -n mayastor-e2e
   kubectl delete -f ${DEPLOYDIR}/workload_monitor/workload_monitor.yaml
   kubectl delete configmap td-config -n mayastor-e2e
+  kubectl delete -f ${DEPLOYDIR}/test_director/test_director_secret.yaml
```
Comment (chriswldenyer) [inline src/tools/extended-test-framework/scripts/deploy.sh:99]: We should remove this line as we won't be creating the secret this way. We can assume the secret is already created on the cluster.

File: src/tools/extended-test-framework/scripts/deploy.sh [hunk 2]
```diff
@@ -107,7 +108,7 @@ else
   PLAN=${PLANARG} envsubst < ${DEPLOYDIR}/test_director/config.yaml.template > $tmpfile
   kubectl create configmap td-config -n mayastor-e2e --from-file=config-local.yaml=${tmpfile}
   rm ${tmpfile}
-
+  kubectl create -f ${DEPLOYDIR}/test_director/test_director_secret.yaml
```
Comment (chriswldenyer) [inline src/tools/extended-test-framework/scripts/deploy.sh:111]: Also this line.

**Review by chriswldenyer — CHANGES_REQUESTED** [review]
Just remove the use of test_director_secret.yaml

**Review by chriswldenyer — APPROVED** [review]
LGTM

**Review by payes — APPROVED** [review]
LGTM

**Review by chriswldenyer — APPROVED** [review]
LGTM

**Pull request merged**